src/panels/config/integrations/ha-integration-header.ts

```diff
@@ -81,6 +81,7 @@ export class HaIntegrationHeader extends LitElement {
       <slot name="above-header"></slot>
       <div class="header">
         <img
+          alt=""
```

Comment:
The primary and secondary text sections of this component do a fine job describing the image.